petastorm/workers_pool/process_pool.py

```diff
@@ -311,6 +311,7 @@ def diagnostics(self):
             'items_consumed': self._ventilated_items,
             'items_produced': self._ventilated_items_processed,
             'items_inprocess': self._ventilated_items - self._ventilated_items_processed,
+            'zmq_copy_buffers': self._zmq_copy_buffers
```

Comment:
I extended the Worker Pool `diagnostics()` rather than read into a chain of private fields in the test.